tpgtools/main.go

```diff
@@ -63,37 +65,39 @@ func main() {
 
 	if mode != nil && *mode == "serialization" {
 		generateSerializationLogic(resourcesForVersion)
-	} else {
-		for _, resource := range resourcesForVersion {
-			resJSON, err := json.MarshalIndent(resource, "", "  ")
-			if err != nil {
-				glog.Errorf("Failed to marshal resource struct")
-			} else {
-				glog.Infof("Generating from resource %s", string(resJSON))
-			}
-			generateResourceFile(resource)
-			generateSweeperFile(resource)
-			// Disabled to allow handwriting files until samples exist
-			// generateResourceWebsiteFile(resource, resources, version)
-		}
+		return
```

Comment:
Doing an early return here will mean we never copy handwritten files in serialization mode, right? I _think_ that's safe if that's the case.

Reply:
I don't know why it's not visible in this diff but [we currently do not copy the handwritten files for serialization mode](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/tpgtools/main.go#L67-L91). Its just not clearly visible in this diff even though the change is there. Maybe it's a github bug

Reply:
Ah- nope, I just misread it. Diffs with indentation changes are hard to read!